lib/_debugger.js

```diff
@@ -4,8 +4,8 @@ const util = require('util');
 const path = require('path');
 const net = require('net');
 const vm = require('vm');
-const module = require('module');
-const repl = module.requireRepl();
+const Module = require('module');
```

Comment:
`Module` is a constructor so it makes sense. It's exactly how it's called in `lib/module.js`